### PR TITLE
Implemented sass and es6 support into project;

### PR DESCRIPTION
**Pull request by dmitry-honcharuk**
Implemented sass and es6 support into project;
Nodemon is ignoring 'src' and 'web' folders

### CODE REVIEW

**Comment by dmitry-honcharuk** [comment]
Resolves #8 issue

File: index.js [hunk 1]
```diff
@@ -48,4 +48,4 @@ app.use('/assets', express.static(path.resolve(`${__dirname}/web`)));
 app.use('/api', apiRouter);
 app.use('/', viewsRouter);
 
-app.listen(process.env.PORT, () => console.log(`Listening on port ${process.env.PORT}`));
+app.listen(process.env.PORT || 3000, () => console.log(`Listening on port ${process.env.PORT || 3000}`));
```
Comment (evgeniyannenkov) [inline index.js]: const PORT = process.env.PORT || 3000;

File: package.json [hunk 2]
```diff
@@ -33,12 +34,21 @@
     "pug": "^2.0.0-rc.2"
```
Comment (evgeniyannenkov) [inline package.json]: Have you still needed pug as a dependency?

File: src/app.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import './styles.scss';
+
+console.log("Scripts!");
```
Comment (evgeniyannenkov) [inline src/app.js]: Please remove debug from repository

**Pull request merged**